### PR TITLE
Updated index.md for 1.18-2 and 1.19-2

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*
Updated for 1.18-2 and 1.19-2 releases

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/contents/index.md [hunk 1]
```diff
@@ -118,5 +118,5 @@ Machine Images), but it does use the EKS Optimized AMI. See the project
 repository for the [EKS Optimized AMI](https://github.com/awslabs/amazon-eks-ami)
 if you are interested in the AL2 container runtime kernel version.
```
Comment (TerryHowe) [inline docs/contents/index.md:120]: Good catch to add these, but somewhere it seems like we should have a complete list of releases including the old. Should this get broken down to a 1-18 and 1-19 section?

Comment (TerryHowe) [reply docs/contents/index.md:120]: Latest first

Comment (TerryHowe) [reply docs/contents/index.md:120]: I created a ticket to enumerate all these various links in the runbook. We should have a ticket for each one so we don't miss them or some sort of checkbox

Comment (kschumy) [reply docs/contents/index.md:120]: > Good catch to add these, but somewhere it seems like we should have a complete list of releases including the old. Should this get broken down to a 1-18 and 1-19 section?

I see what you mean. Do you want me to make that change? I think this is one of those things that ends up on the website. Do we want all the releases on there?

Comment (TerryHowe) [reply docs/contents/index.md:120]: Yes. I have no better idea right now, it seems like all the old releases should be listed somewhere.

File: docs/contents/index.md [hunk 2]
```diff
@@ -118,5 +118,5 @@ Machine Images), but it does use the EKS Optimized AMI. See the project
...
 
-* [EKS-D v1-18-eks-1 Version Dependencies](releases/1-18/1/index.md)
-* [EKS-D v1-19-eks-1 Version Dependencies](releases/1-19/1/index.md)
+* [EKS-D v1-18-eks-2 Version Dependencies](releases/1-18/2/index.md)
+* [EKS-D v1-19-eks-2 Version Dependencies](releases/1-19/2/index.md)
```
Comment (TerryHowe) [inline docs/contents/index.md:122]:
```suggestion
* [EKS-D v1-19-eks-2 Version Dependencies](releases/1-19/2/index.md)
* [EKS-D v1-19-eks-1 Version Dependencies](releases/1-19/1/index.md)
* [EKS-D v1-18-eks-2 Version Dependencies](releases/1-18/2/index.md)
* [EKS-D v1-18-eks-1 Version Dependencies](releases/1-18/1/index.md)
```

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Pull request merged**